README.md

```diff
@@ -1,22 +1,41 @@
-# orca-recipes-airflow
-This repo will serve as a prototype for basic airflow for the DPE team
+# ORCA Airflow Recipes
 
-## start up
-To run airflow, you will need Docker and Docker-Compose installed on your machine.
+This repository contains Airflow recipes (DAGs) for data processing and engineering at Sage Bionetworks.
 
-Once you have the prerequisites installed, you simply need to navigate into the `orca-recipes-airflow` directory and run the following commands.
+## Quick Start
 
-### Build Docker Containers
-```
-docker-compose build
+This assumes that you have Docker installed with [Docker Compose V2](https://docs.docker.com/compose/compose-v2/). It's recommended that you leverage the included Dev Container definition (_i.e._ `devcontainer.json`) to standardize your development environment. You can use the [Dev Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) VS Code extension or GitHub Codespaces.
+
+```console
+# Duplicate example `.env` file and edit as needed
+cp .env.example .env
+docker compose up --build --detach
 ```
 
-### Start Docker Containers
+After you're done editing your DAG, you can restart the containers so Airflow can pick up on the latest version as follows.
+
+```console
+docker compose restart
 ```
-docker-compose up
+
+If you edit `Dockerfile`, `docker-compose.yaml`, or `Pipfile`/`Pipfile.lock`, you'll need to rebuild the containers as follows. 
```

Comment:
editing `airflow.config` and any sort of `.env` file will require a rebuild as well

Reply:
Good catch!